testsuite/unittests/test_aeif_cond_beta_multisynapse.sli

```diff
@@ -75,12 +75,14 @@ if
 
 /tolerance   	5e-2 def %mV
 /simulation_t	2500 def %ms
-/V_peak			0. def
-/a 				4. def
-/b 				80.5 def
+/V_peak		0. def
+/a 		4. def
+/b 		80.5 def
+/num_of_receptors       3 def
+/E_rev		        [0.0 0.0 -85.0] def % synaptic reversal potentials  
 /taus_decay		[400.0 200.0 200.0] def % synaptic decay times  
 /taus_rise		[200.0 100.0 100.0] def % synaptic rise times  
-/weights		[0.1 0.1 -0.1] def % synaptic weights
+/weights		[0.1 0.1 0.1] def % synaptic weights
```

Comment:
Wouldn't it be better to make all reversal potentials, time constants and weights different from each other to make sure nothing gets mixed up in the code?